src/components/Button/Button.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'; 
+import PropTypes from 'prop-types'; 
+
+const Button = ({ onClick }) => (
```

Comment:
Passing 'handleClick' instead of onClick help differentiate between a func that you wrote and the built-in button method 'onClick'

Reply:
Thanks for this feedback